scripts/zones/Port_Bastok/npcs/Festive_Moogle.lua

```diff
@@ -5,17 +5,33 @@
 --  !pos 46.902 8.499 -242.276 236
 -----------------------------------
 require("scripts/globals/npc_util")
+-----------------------------------
 
-    local festiveItems = {
-        [1] = function(player) return (player:getCharVar("festiveMoogleNomadCap")) end,
-        [2] = function(player) return (player:getCharVar("festiveMoogleMoogleCap")) end,
-        [3] = function(player) return (player:getCharVar("festiveMoogleMoogleRod")) end,
-        [4] = function(player) return (player:getCharVar("festiveMoogleHarpsichord")) end,
-        [5] = function(player) return (player:getCharVar("festiveMooglestuffedChocobo")) end,
-        [6] = function(player) return (player:getCharVar("festiveMoogleTidalTalisman")) end,
-        [7] = function(player) return (player:getCharVar("festiveMoogleDestrierBeret")) end,
-        [8] = function(player) return (player:getCharVar("festiveMoogleChocoboShurt")) end,
-    }
+function onTrade(player,npc,trade)
+    --TODO: trade of pellet for prize
```

Comment:
its called a "pell". not pellet.

Green Mog Pell
Red Mog Pell
Silver Mog Pell
Gold Mog Pell

etc etc

Reply:
got it, will rename it to pells =)